src/crud.py

```
@@ -35,6 +36,92 @@
+                current_batch_texts.append(text)
+                current_batch_tokens += text_tokens
+
+        # Append remaining batch
+        if current_batch_ids:
+            batches.append((current_batch_ids, current_batch_texts))
+
+        # Process all batches
+        all_embeddings: dict[str, list[float]] = {}
+        for batch_ids, batch_texts in batches:
+            response = await self.client.embeddings.create(
+                model="text-embedding-3-small", input=batch_texts
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)